.mocharc.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
     "source-map-support/register"
   ],
   retries: 0,
+  spec: 'test/**/*_test.ts',
```

Comment:
@treefitty are you sure this is required?
Because the recursive and extension configuration should cover the `**/*.[ts,js]` and mocha searches tests files in the `test` folder by default.
Or has it something to do with the test coverage?